src/components/calcite-input-message/calcite-input-message.scss

```diff
@@ -0,0 +1,112 @@
+// theme variables
+// light theme
+:host {
+  --calcite-input-message-floating-background: #{rgba($ui-foreground, 0.96)};
+}
+
+// dark theme
+:host([theme="dark"]) {
+  --calcite-input-message-floating-background: #{rgba($ui-foreground-hover-dark, 0.96)};
+}
+
+:host {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  text-align: left;
+  color: var(--calcite-ui-text-2);
+  font-weight: 400;
+  @include font-size(-3);
+  transition: $transition;
+  visibility: hidden;
+  opacity: 0;
+  height: 0;
+  margin-top: 0;
+}
+
+:host([active]) {
+  opacity: 1;
+  margin-top: $baseline/4;
+  height: auto;
+  max-height: 100vh;
+  visibility: visible;
+}
+
+:host([type="floating"]) {
+  padding: $baseline/2;
+  transform: translate3d(0, -$baseline, 0);
+  //todo
+  background-color: var(--calcite-input-message-floating-background);
+  position: absolute;
+  width: 100%;
+  top: 100%;
+  left: 0;
+  right: 0;
+  height: auto;
+  margin-top: 0;
+  border-radius: var(--calcite-border-radius);
+  box-shadow: $shadow-2;
+  z-index: 101;
+}
+
+:host([type="floating"][active]) {
+  transform: translate3d(0, 0, 0);
+  margin: 0;
+}
+
+//todo rtl
+.calcite-input-message-icon {
```

Comment:
This component actually has `shadow: false` because shadow DOM isn't parsable by password managers, so using `:host` here doesn't work to style off of.